pkg/virtctl/create/instancetype/instancetype_test.go

```diff
@@ -141,16 +169,20 @@ var _ = Describe("create instancetype", func() {
 			Entry("VirtualMachineClusterPreference", false),
 		)
 
-		DescribeTable("with defined gpus", func(extraArgs ...string) {
+		runDeviceCmd := func(flagName, deviceParam string, extraArgs ...string) *instancetypev1beta1.VirtualMachineInstancetypeSpec {
```

Comment:
Can you move this down to L228 after `runCmd` and make it a regular func instead of a lambda?